src/js/pages/services/ServicesTab.js

```diff
@@ -210,10 +226,11 @@ var ServicesTab = React.createClass({
             <ServiceSearchFilter
               handleFilterChange={this.handleFilterChange} />
             <button className="button button-stroke button-inverse"
-                    onClick={this.handleOpenGroupFormModal}>
+              onClick={() => this.handleOpenModal(SERVICE_GROUP_FORM_MODAL)}>
               Create Group
             </button>
-            <button className="button button-success">
+            <button className="button button-success"
+              onClick={() => this.handleOpenModal(SERVICE_FORM_MODAL)}>
```

Comment:
I hate to do this but I think I heard somewhere that we are supposed to use the METHODS_TO_BIND way of doing things consistently...